sale_automatic_workflow/models/sale_order.py

```diff
@@ -18,23 +17,13 @@ class SaleOrder(models.Model):
     all_qty_delivered = fields.Boolean(
         compute="_compute_all_qty_delivered",
         string="All quantities delivered",
-        store=True,
+        store=False,
```

Comment:
I don't think you should change this now because you could break existing installations. If you do it, at least, add a search method which logs a deprecation warning.

What do you think?

Reply:
@PierrickBrun @sebastienbeau ping

Reply:
Hi @simahawk I think it may be better to just let it stored and add a comment & roadmap message to make it non-stored in a future version

Reply:
Salut! 
As you prefer. So, please:

1. restore this flag
2. add a comment here and add a note to roadmap
3. squash the last commits of fixes done via GH on top of the migration commit
4. reword the commit for sale_automatic_workflow as this: `sale_automatic_workflow: depend on sale_delivery_state`

Merci! :)